.docker/source/Dockerfile

```diff
@@ -4,35 +4,26 @@
 FROM moveit/moveit:melodic-ci-shadow-fixed
 MAINTAINER Dave Coleman dave@picknik.ai
 
-ENV CATKIN_WS=/root/ws_moveit
-RUN mkdir -p $CATKIN_WS/src
-WORKDIR $CATKIN_WS/src
-
-# Download moveit source
-RUN wstool init . && \
-    wstool merge https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
-    wstool update
-
-# Update apt-get because osrf image clears this cache. download deps
-# Note that because we're building on top of melodic-ci, there should not be any deps installed
-# unless something has changed in the source code since the other container was made
-# (they are triggered together so should only be one-build out of sync)
-RUN apt-get -qq update && \
-    apt-get -qq install -y \
-        wget && \
-    rosdep update && \
-    rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} --as-root=apt:false && \
-    rm -rf /var/lib/apt/lists/*
-
-# Replacing shell with bash for later docker build commands
-RUN mv /bin/sh /bin/sh-old && \
-    ln -s /bin/bash /bin/sh
-
-# Build repo
-WORKDIR $CATKIN_WS
-ENV TERM xterm
```

Comment:
this removes various warnings and errors in the CI build...

Reply:
It's redundant, because these are set by the previous docker image already.